homeassistant/components/sonos/entity.py

```diff
@@ -57,8 +57,11 @@ async def async_will_remove_from_hass(self) -> None:
     async def async_fallback_poll(self, now: datetime.datetime) -> None:
         """Poll the entity if subscriptions fail."""
         if not self.speaker.subscriptions_failed:
+            listener_msg: str
             if soco_config.EVENT_ADVERTISE_IP:
-                listener_msg = f"{self.speaker.subscription_address} (advertising as {soco_config.EVENT_ADVERTISE_IP})"
+                listener_msg = (
+                    f"{listener_msg} (advertising as {soco_config.EVENT_ADVERTISE_IP})"
+                )
```

Comment:
I don't think the changes here are needed anymore. VS Code even highlights a potential unbound variable for `listener_msg` in the f-string. Try reverting these lines, that should work.